Buildings/Experimental/OpenBuildingControl/CDL/Continuous/Validation/MovingMean.mo

```diff
@@ -0,0 +1,59 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Continuous.Validation;
+model MovingMean "Validation model for the MovingMean block"
+extends Modelica.Icons.Example;
+
+  Modelica.Blocks.Noise.UniformNoise uniformNoise(
```

Comment:
Please use another input signal, as `uniformNoise` is difficult for others to reproduce when they want to validate their implementation.